dspace/config/clarin-dspace.cfg

```diff
@@ -144,3 +144,12 @@ matomo.site.id = 1
 matomo.tracker.bitstream.site_id = 1
 matomo.tracker.oai.site_id = 1
 statistics.cache-server.uri = http://cache-server.none
+
+##### Citacepro config #####
+rest.properties.exposed= citace.pro.url
+citace.pro.url = https://www.citacepro.com/api/dspace/citace/oai
+citace.pro.url = dspace.tul.cz
+
+rest.properties.exposed= citace.pro.status
```

Comment:
Please add this property into `rest.properties.exposed` in the line 57 in the `clarin-dspace.cfg`. And please remove this repeatable property definition.